src/App.js

```diff
@@ -2,41 +2,35 @@ import React from 'react';
 import './style.css';
 import Buttonpanel from './Components/buttonpanel';
 import Display from './Components/display';
-import calculate from './Logic/calculate';
-import calculator from './Logic/calculate';
+import Calculator from './Logic/calculate';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { result: '' };
+    this.state = { result: '', calculator: new Calculator() };
+    this.handleClick = this.handleClick.bind(this);
   }
```

Comment:
`STATE` was asked to be initialised with these 3 properties --> **total, next, operation** [[see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/stateful.png)]

Reply:
🚀